Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/IndexSearch.cs

```diff
@@ -92,8 +95,41 @@ internal async static Task<List<Result>> ExecuteWindowsIndexSearchAsync(string i
                 return new List<Result>();
 
             var constructedQuery = constructQuery(searchString);
-            return await ExecuteWindowsIndexSearchAsync(constructedQuery, connectionString, query, token);
+            return await RemoveResultsInExclusionList(
+                        await ExecuteWindowsIndexSearchAsync(constructedQuery, connectionString, query, token),
```

Comment:
@taooceros should ConfigureAwait(false) and cancellation token be added for the two methods?;

Reply:
Ah you are right!